tests/test.py

```diff
@@ -271,7 +271,7 @@ def test_manifest_static_storage_file(self, web_container):
         """
         hashed_svg = web_container.exec_find(
             ['static/admin/img', '-regextype', 'posix-egrep', '-regex',
-             '.*\.[a-f0-9]{12}\.svg$'])
+             r'.*\.[a-f0-9]{12}\.svg$'])
```

Comment:
Technically this isn't a regex for Python but... I think this is better than double-escaping things?

Reply:
`r""` strings may have been designed for regexes, but they're documented as generic unescaped strings with an implementation quirk that you can't end on a trailing `\`.